src/imitation/algorithms/bc.py

```diff
@@ -424,15 +424,17 @@ def _on_epoch_end(epoch_number: int):
             _on_epoch_end,
         )
         batches_with_stats = enumerate_batches(demonstration_batches)
+        tqdm_progress_bar: tqdm.tqdm = None
```

Comment:
Unless I'm misunderstanding the `tqdm.tqdm` type this shouldn't type check.

```suggestion
        tqdm_progress_bar: Optional[tqdm.tqdm] = None
```